docs/topics/multiplatform-mobile/multiplatform-mobile-getting-started.md

```diff
@@ -16,41 +16,38 @@ Kotlin Multiplatform Mobile (KMM) is an SDK designed to simplify the development
 You can share common code between iOS and Android apps and write platform-specific code only where it's necessary.
 For example, to implement a native UI or when working with platform-specific APIs.
 
-Watch the introductory [video](https://www.youtube.com/watch?v=mdN6P6RI__k), in which Kotlin Product Marketing Manager Ekaterina 
-Petrova explains what Kotlin Multiplatform Mobile is and how to use it in your project. With Ekaterina,
-you'll set up an environment and prepare to create your first cross-platform mobile application:
+> You can also watch the [introductory video](https://www.youtube.com/watch?v=GcqFhoUuNNI) with a general explanation on
+> what Kotlin Multiplatform Mobile is and how to use it in your project.
+>
+{type="tip"}
+
+Get to know Kotlin Multiplatform Mobile and complete these steps to create a mobile app the works on both Android and iOS:
+
+![First step](icon-1.svg){width=25}{type="joined"} [Set up environment for cross-platform mobile development](multiplatform-mobile-setup.md)
 
-<video href="mdN6P6RI__k" title="Meet Kotlin Multiplatform Mobile!"/>
+![Second step](icon-2.svg){width=25}{type="joined"} [Create your first app that works both on Android and iOS with IDE](multiplatform-mobile-create-first-app.md)
 
-You can also check out other videos about [Kotlin Multiplatform Multiverse](https://www.youtube.com/playlist?list=PLlFc5cFwUnmy_oVc9YQzjasSNoAk4hk_C) on YouTube.
+![Third step](icon-3.svg){width=25}{type="joined"} [Add dependencies to your project](multiplatform-mobile-dependencies.md)
 
-## Supported platforms
+![Fourth step](icon-4.svg){width=25}{type="joined"} [Upgrade your app](multiplatform-mobile-upgrade-app.md)
 
-* Android applications and libraries
-* [Android NDK](https://developer.android.com/ndk) (ARM64 and ARM32)
-* Apple iOS devices (ARM64 and ARM32) and simulators
-* Apple watchOS devices (ARM64 and ARM32) and simulators
+![Fifth step](icon-5.svg){width=25}{type="joined"} [Wrap up your project](multiplatform-mobile-wrap-up.md)
 
-[Kotlin Multiplatform](multiplatform.md) technology also supports [other platforms](multiplatform-dsl-reference.md#targets), including
-JavaScript, Linux, Windows, and WebAssembly.
+## Next step
 
-## Start from scratch
+Start by [setting up an environment](multiplatform-mobile-setup.md) for Kotlin Multiplatform Mobile development.
 
-* [Set up the environment for cross-platform mobile development](multiplatform-mobile-setup.md)
-* [Create your first app that works both on Android and iOS with IDE](multiplatform-mobile-create-first-app.md)
-* [Check out the list of sample projects](multiplatform-mobile-samples.md)
-* [Introduce cross-platform mobile development to your team](multiplatform-mobile-introduce-your-team.md)
+### See also
 
-## Make your Android application work on iOS
+If you already have experience with mobile development, or you have an existing Android application, you can make it work
```

Comment:
It might be a bit confusing which path to follow: almost all our target audience for these guides have experience with mobile development, or have an existing Android application :) Nevertheless, it’s better for them to use the first “path”. I’d put it more explicitly, like: 

"If you want to convert your existing Android application into a multiplatform one, check how you can make it work on iOS:"